tests/unit/processing/test_extract_metadata.py

```diff
@@ -16,6 +16,13 @@ def setUp(self):
             self.business_logic, self.uri_provider, self.s3_provider, self.downloader, self.schema_validator
         )
 
+        def mock_config_fn(name):
+            if name == "schema_4_feature_flag":
+                return "True"
+
+        mock_config = patch("backend.common.corpora_config.CorporaConfig.__getattr__", side_effect=mock_config_fn)
+        mock_config.start()
```

Comment:
add a tear down for this mock

Reply:
done! went through and added it to some other test classes missing them as well